packages/mdc-ripple/foundation.js

```diff
@@ -188,6 +177,18 @@ class MDCRippleFoundation extends MDCFoundation {
   }
 
   /**
+   * We compute this property so that we are not querying information about the client
+   * until the point in time where the foundation requests it. This prevents scenarios where
+   * client-side feature-detection may happen too early, such as when components are rendered on the server
+   * and then initialized at mount time on the client.
+   * @return {boolean}
+   * @private
+   */
+  isSupported_() {
```

Comment:
I thought we were going to examine making this a static getter? But given that it's internal I can go either way on it.

Reply:
I thought the intent is to have implementer be able to override this via adapter. As such it cannot be static method - needs instance. If the intent is to have this always call util.browserSupportsCssVars, we can do it from here. Let me know what you think.

Reply:
Oh because the adapter is connected to the instance... yup, that would be why I thought I remembered there being a reason why we couldn't do that.